cms/src/test/resources/test-data/api-specifications/ApiSpecSyncFromApigeeJobIntegrationTest/oasV3_complete.html

```diff
@@ -20,185 +20,156 @@
+                                    </a>
+                                </li>
+                                <li class="nhsd-m-sticky-navigation__item nhsd-m-sticky-navigation__item--nested" data-nav-content="api-Patients-get-patient">
+                                    <a class="nhsd-a-link" href="#api-Patients-get-patient">
+                                        Retrieve a patient's resource
+                                    </a>
+                                </li>
+                                <li class="nhsd-m-sticky-navigation__item nhsd-m-sticky-navigation__item--nested" data-nav-content="api-Patients-search-patient">
+                                    <a class="nhsd-a-link" href="#api-Patients-search-patient">
+                                        Search for patient
+                                    </a>
+                                </li>
+                                <li class="nhsd-m-sticky-navigation__item nhsd-m-sticky-navigation__item--nested" data-nav-content="api-Patients-update-patient-partial">
+                                    <a class="nhsd-a-link" href="#api-Patients-update-patient-partial">
+                                        Update a patient's resource
+                                    </a>
+                                </li>
+                                <li class="nhsd-m-sticky-navigation__item" data-nav-content="api-Polling">
+                                    <a class="nhsd-a-link" href="#api-Polling">
+                                        Endpoints: Polling
+                                    </a>
+                                </li>
+                                <li class="nhsd-m-sticky-navigation__item nhsd-m-sticky-navigation__item--nested" data-nav-content="api-Polling-polling">
+                                    <a class="nhsd-a-link" href="#api-Polling-polling">
+                                        Retrieve the outcome of the accepted update.
+                                    </a>
+                                </li>
             </ul>
         </nav>
-    </div>
-    <!-- end sticky-nav -->
 </div>
-
+<!-- end sticky-nav -->
+</div>
 <div id="content" class="nhsd-t-col-xs-12 nhsd-t-col-s-8" aria-label="Document content">
-
     <!-- API Title: Personal Demographics Service (FHIR) API -->
-
     <div id="api-description" class="article-section">
         <h2 id="api-description__overview">Overview</h2>
-        <p>Use this API to access the <a href="https://digital.nhs.uk/services/demographics">Personal Demographics Service (PDS)</a></p>
-        <h2 id="api-description__legal-use">Legal use</h2>
-        <p>This API can only be used where there is a legal basis.
-    </div>
-
-    <div id="api-Patients" class="article-section">
-        <h2>Endpoints: Patients</h2>
+<p>Use this API to access the <a href="https://digital.nhs.uk/services/demographics">Personal Demographics Service (PDS)</a></p>
+<h2 id="api-description__legal-use">Legal use</h2>
```

Comment:
Headers should use an appropriate header class, eg, `nhsd-t-heading-l`.

Reply:
Addressed in [ADZ-1191](https://nhsd-jira.digital.nhs.uk/browse/ADZ-1191) / #1591.